template.yaml

```diff
@@ -31,6 +31,10 @@ Parameters:
     Description: "The ARN of the permissions boundary to apply when creating IAM roles"
     Type: String
     Default: "none"
+  MaxContainerCount:
+    Description: Maximum number of FE node/express container tasks to allow autoscaling to reach
+    Type: Number
+    Default: 12
```

Comment:
Is this 12 independent of the journey? f2f will not have the same traffic as other journeys.

Reply:
Leave max container count with 12